File: java/src/main/java/mx/iteso/adapter/BackgroundImage.java
package mx.iteso.adapter;

/**
 * Background image Adapter.
 */
public class BackgroundImage {
    /**
     * new Image.
     */
    private Image image;

    /**
     * Background image.
     * @param background .
     */
    public BackgroundImage(final Image background) {
        this.image = new Image("image.jpg");
    }

    /**
     *
     * @return img
     */
    public Image getImage() {
        return this.image;
    }
}

File: csharp/AdapterPractice/AdapterPractice/Adapter/BackgroundImage.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AdapterPractice.Adapter
{
    public class BackgroundImage
    {
      
        private Image image;

        public BackgroundImage( Image background)
        {
            this.image = new Image("image.jpg");
        }

        public Image getImage()
        {
            return this.image;
        }
    }
}
